company/values.md

```diff
@@ -4,20 +4,27 @@
 
 [effective-oxford]: https://www.oxfordlearnersdictionaries.com/definition/english/effective
 
-Being effective is split up in two parts; producing results and communication
+Being effective is split up in two parts; producing ***results*** and ***communication***
 about what our goals are.
 
 ### Results
 
-We value results over measuring hours spent working. We do not dictate how
-someone should schedule their day - we trust individuals to do what is right.
-Everyone should feel empowered to do the work needed to achieve our goals.
+#### Results not Hours
+
+We care about what you achieve: the code you shipped, the user you made happy,
+and the team member you helped. We value results over measuring hours spent
+working. We do not dictate how someone should schedule their day - we trust
+individuals to do what is right. Everyone should feel empowered to do the work
+needed to achieve our goals.
 
 There may be times when more is needed to meet a goal, deadline or customer
 situation. We take the responsibility to ensure they are the exceptions rather
 than the rule.
 
-#### Bias for action
+Do not incite competition by proclaiming how many hours you worked yesterday.
+If you are working too many hours, talk to your manager to discuss solutions.
```

Comment:
I understand what you're trying to get at - and support the intent - but I'm not sure this is quite the way I would express it. I prefer framing the values as what 'we' do, not what 'you' should (or should not) do.

```suggestion
We don't try to compete over the hours we work - everyone does what is right for them.  We focus on what gets done and take pride in seeing each other succeed.
```

Reply:
like that change. The language I'd written was taken straight out of the GitLab handbook, happy to amend.

Reply:
@joepavitt Can you apply that suggestion, and I'll merge this PR after?

Reply:
done